### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ShinazBinShajahan**

### CODE REVIEW

**Comment by ashishshaiju** [comment]
Move the printer.discord.com gif to the last divider line. 
And also remove the underline from the subscribe and support heading.

File: README.md [hunk 1]
```diff
@@ -5,14 +5,7 @@
 <a href="https://reactjs.org/">
 <img align="right" src="https://github.com/ashishshaiju/ashishshaiju/blob/files/images/Coding.png" alt="coding ?"> </a>
 
-### 💖 Subscribe And Support
```
Comment (ashishshaiju) [inline README.md:8]: Remove the underline from the subscribe and support heading.

Comment (ShinazBinShajahan) [reply README.md:8]: Yes. will do

**Review by ShinazBinShajahan — COMMENTED** [review]
Okay

**Review by ashishshaiju — CHANGES_REQUESTED** [review]
Also move the printer.discord.com gif to the last divider line.

**Pull request closed without merging**